sections/main-product.liquid

```diff
@@ -9,6 +9,9 @@
   {{ 'component-slider.css' | asset_url | stylesheet_tag }}
   {{ 'component-rating.css' | asset_url | stylesheet_tag }}
   {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+  {{ 'component-product-variant-picker.css' | asset_url | stylesheet_tag }}
+  {{ 'component-swatch-input.css' | asset_url | stylesheet_tag }}
+  {{ 'component-swatch.css' | asset_url | stylesheet_tag }}
```

Comment:
I'm wondering if we should actually load this CSS within the snippet itself 🤔  the `component-swatch.css`. Since it's specific to that one snippet if I'm not mistaking.

Reply:
As discussed on Slack, we don't want to move it inside the snippet because it will try to fetch this CSS file for every instance of a swatch-input (or swatch) instead of once per page